### ISSUE TITLE
Container shouldn't run as root

### ISSUE THREAD

**Issue by dza89** [SebDanielsson/docker-wireguard-transmission#8]
It should be able to copy most from here to get this working
https://github.com/haugene/docker-transmission-openvpn

**Comment by mrsufgi** [comment]
Or just contribute wireguard support to docker-transmission-openvpn and together create docker-transmission-vpn :)

**Comment by SebDanielsson** [comment]
I read a blog post about not running containers as root. The person suggested S6 overlay. Do you have any experience using it?

**Comment by vidurb** [comment]
Hi,
I wrote a [PR](https://github.com/SebDanielsson/docker-wireguard-transmission/pull/9) implementing S6 overlay so as to allow running as a different user easily (there are other methods, but I chose this as many of the `linuxserver` images use S6 overlay so I took that as a sign of reliability). I copied some of the code Haugene uses in their OpenVPN Transmission image.
The code is pulled from my [fork](https://github.com/vidurb/docker-transmission-wireguard) of Seb's image, but unfortunately I'm away from my computer and although I managed to create the PR from a mobile device I can't test the images. I should be able to do so in a few days, but if anyone wants to take the initiative, test builds are on [Docker Hub](https://hub.docker.com/repository/docker/vidurb/transmission-wireguard-test).
